flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/JobGraphConfigurationUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.jobgraph;
+
+import org.apache.flink.configuration.SchedulerExecutionMode;
+import org.apache.flink.runtime.state.KeyGroupRangeAssignment;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Utilities for configuring {@link JobGraph JobGraphs} before their execution. */
+public final class JobGraphConfigurationUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(JobGraphConfigurationUtils.class);
+
+    /**
+     * Configures a {@link JobGraph} for the reactive {@link SchedulerExecutionMode}.
+     *
+     * @param jobGraph The JobGraph.
+     */
+    public static void configureJobGraphForReactiveMode(JobGraph jobGraph) {
```

Comment:
Thanks a lot for adding a test for this. I don't know why I forgot to add one :cry: